klayout/drc/rule_decks/mim_a.drc

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if MIM_OPTION == 'A'
+  #================================================
+  #------------MIM CAPACITOR OPTION A -------------
+  #================================================
+
+  logger.info('Starting MIM Capacitor Option A derivations')
+
+  m2_fusetop = metal2.interacting(fusetop)
+  mim_virtual = fusetop.sized(1.06.um).and(m2_fusetop)
+
+  # Rule MIM.1: Minimum MiM bottom plate spacing to the bottom plate metal
+  ## (whether adjacent MiM or routing metal) is 1.2µm.
+  logger.info('Executing rule MIM.1')
+  mim1_l1 = metal2.separation(mim_virtual, transparent, 1.2.um)
+  mim1_l1.output('MIM.1', 'MIM.1 : Minimum MiM bottom plate spacing to the bottom plate metal
+                 (whether adjacent MiM or routing metal): 1.2µm')
+  mim1_l1.forget
+
+  # Rule MIM.2: Minimum MiM bottom plate overlap of Via2 layer.
+  ## [This is applicable for via2 within 1.06um oversize of FuseTop layer
+  ## (referenced to virtual bottom plate)] is 0.4µm.
+  logger.info('Executing rule MIM.2')
+  mim2_via = via2.overlapping(mim_virtual)
+  mim2_l1 = mim2_via.enclosed(metal2, 0.4.um, euclidian).polygons(0.001.um)
+  mim2_l2 = mim2_via.not_outside(metal2).not(metal2)
```

Comment:
can you add a comment describe this line (is that for selecting the bottom plate?), because I can't really relate this to the current wording of the rule.